aopy/preproc/neuropixel.py

```diff
@@ -3,6 +3,7 @@
 from ..data import load_neuropixel_data, load_ks_output, get_channel_bank_name
 import os
 import glob
+from ibldsp.voltage import destripe_lfp

```

Comment:
Do we need to add idldsp to setup.py? (and mock imports in the docs)

Reply:
I added ibllib in setup.py and conf.py in docs. Is this enough?

Reply:
yup I think its good!